README.md

```diff
@@ -78,12 +81,15 @@ Once the pacoloco server is up and running it is time to configure the user host
 
 ```conf
 [core]
+Include = /etc/pacman.d/pacoloco-mirrorlist
```

Comment:
It is easier if we use only one Include here. If someone wants to use reflector then the main file `/etc/pacman.d/mirrorlist` can be generated/used for pacoloco.

Reverting the example back to 1-Include version makes it simpler.

Reply:
If you prefer, I'll do it. Personally I had found myself the mirrorlist getting replaced by reflector and the thing I missed was that I couldn't use both reflector and pacoloco (pacoloco at home, reflector when pacoloco wasn't available), unless by doing a custom configuration on either reflector or pacoloco

Reply:
It is fine to have this ` /etc/pacman.d/pacoloco-mirrorlist` for those who need it (like in your case). But for the documentation it would be better to keep it simpler. You can add a note after the config file saying that it is possible to use both reflector and non-reflector mirrorlists together.

Reply:
Agree, I'll edit it